packages/contracts-bedrock/deploy-config/goerli.json

```diff
@@ -1,26 +1,39 @@
 {
-  "l1StartingBlockTag": "0xafce66a0a2446856112e4069b275ad32b1f4a607888f9c4c59eddf9be81f8670",
+  "finalSystemOwner": "DUMMY",
+  "controller": "DUMMY",
+
+  "l1StartingBlockTag": "DUMMY",
   "l1ChainID": 5,
-  "l2ChainID": 111,
+  "l2ChainID": 420,
   "l2BlockTime": 2,
 
-  "maxSequencerDrift": 1000,
-  "sequencerWindowSize": 120,
+  "maxSequencerDrift": 3600,
+  "sequencerWindowSize": 7200,
   "channelTimeout": 120,
-  "p2pSequencerAddress": "0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc",
-  "batchInboxAddress": "0xff00000000000000000000000000000000000002",
-  "batchSenderAddress": "0xa11d2b908470e17923fff184d48269bebbd9b2a5",
-
-  "systemConfigOwner": "0xa0Ee7A142d267C1f36714E4a8F75612F20a79720",
 
-  "l2OutputOracleSubmissionInterval": 6,
+  "p2pSequencerAddress": "DUMMY",
+  "batchInboxAddress": "0xff00000000000000000000000000000000000000",
```

Comment:
This address can be anything - are we OK with making this `0xff00000000000000000000000000000000000000`?

Reply:
While it can be anything, I do think it should be consistent across systems. The monorepo has quite a mix of them in different configs, including: 

```
0xff00000000000000000000000000000000000002
0xff00000000000000000000000000000000000000
0x42000000000000000000000000000000000000ff
```

To minimize bikeshedding here, I'll strawman propose that we go with `0x42000000000000000000000000000000000000ff`, since I like keeping the `0x42` prefix as our calling card.

Reply:
> I do think it should be consistent across systems

... or is it better to make them always unique as replay protection?

Reply:
I like keeping them different per network. How about we just use the chain ID?

Reply:
Chain ID would have to be the prefix to avoid precompiles. But that SGTM.

Reply:
Put the chain ID at the end and went with `0xff00000000000000000000000000000000000420` to disambiguate from other `0x42` addresses.